README.md

```diff
@@ -10,7 +29,52 @@ curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spect
 Set-ExecutionPolicy Bypass -Scope Process -Force; iex  ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/fossas/spectrometer/master/install.ps1'))
 ```
 
-## Building
+**NOTE**: In order to use the executable, you may need to add it to your PATH.
+The installer will report the installation path of the executable to make this
+easier.
+
+## Basic Usage
+
+The tool requires little-to-no configuration to run on its own.
+
+```sh
+fossa2 analyze --output # Analyze the current working directory
+# OR
+fossa2 analyze --output --basedir /path/to/project  # Analyze a specific directory.
+```
+
+The `--output` flag tells the analyzer not to connect to a FOSSA server,
+instead printing out the analysis result to the console.
+
+While we're on the subject, you can (and should!) connect your output directly
+to your FOSSA server!  You can do this by simply supplying the FOSSA API key
```

Comment:
[nit] I would change "your" to "the" to avoid confusion around users thinking they need to run their own FOSSA server.